scripts/vae_paramsweep.py

```diff
@@ -29,60 +31,96 @@
 
 
 def get_param(param):
-    sweep = parameter_df.loc[param, 'sweep']
+    try:
```

Comment:
If you are going to do a try clause here, you might want to leave a message in the except block to let the user know an exception has occurred. Quietly handling this is fine, but will it break your code?

Reply:
good point. I will add in the next commit